app/views/clinicfinders/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= bootstrap_form_tag url: clinicfinder_path do |f| %>
```

Comment:
Somewhat tempted to dump this into `clinicfinders#_new` to make it a bit more restful, and render that partial on page load instead of summoning it thru ajax. Going to think on this.